### DISCUSSION TITLE
How to check changes made to a file in previous merged PR, while a new PR is also merged on top of that ?

### DISCUSSION THREAD

**Discussion by sheikhasim** [tj-actions/changed-files#1414] [Q&A]
Suppose I have a file 'sb1.graphql'. In first PR there are changes made to the file and the PR is merged. 
In second PR , there are changes made to other files , not to the 'sb1.graphql'.
I want to trigger a deployment workflow manually, which has the 'changed-files' action with 'file_path: sb1.graphql'. 
Since first PR has introduced changes to the 'sb1.graphql' file, the action should flag changes as true. But it is giving false , which is maybe expected with the way action is setup.
Is there any workaround for this scenario ? 

Please feel free to ask more details , if the title and question isn't enough.
Thank you.

**Comment by jackton1** [comment]
Hi @sheikhasim Can you provide more details on what you have working right now preferably including the event and description of the PR's i.e 

PR1: 
-  base_branch: ...
-  changed_files: ...

PR2:
-  base_branch: ...
-  changed_files: ...

**Comment by jackton1** [comment]
Since there hasn’t been any response. I’ll close this discussion for now
